ssl/statem/extensions.c

```diff
@@ -10,6 +10,7 @@
 #include <string.h>
 #include "../ssl_locl.h"
 #include "statem_locl.h"
+#include "../record/record_locl.h"
```

Comment:
To call `ssl3_release_write_buffer(ssl)` and `ssl3_setup_write_buffer(ssl, pipes, 0)`